utility/string-util.js

```diff
@@ -53,6 +53,23 @@ const extractAndJoinFields = (data, fieldsToExtract, splitter = ",") => {
     );
 }
 
+async function getSubmisssionRootPath (submission, organizationService) {
+    let rootPath = submission.rootPath;
+    if(!rootPath){
```

Comment:
We should throw an error if a submission doesn't have a rootPath. Because that's definitely a bug, if we infer a rootPath here, we'll potentially cover up a bug and it will come back to bite us from somewhere else eventually.

Reply:
Ok